packages/databricks-vscode/src/configuration/ui/BundleTargetComponent.ts

```diff
@@ -22,6 +22,9 @@ export class BundleTargetComponent extends BaseComponent {
         this.disposables.push(
             this.configModel.onDidChangeTarget(() => {
                 this.onDidChangeEmitter.fire();
+            }),
+            this.configModel.bundlePreValidateModel.onDidChange(() => {
```

Comment:
Can we avoid exposing sub-models from `configModel`? From the maintenance perspective it might make our life easier if we were to keep them as an implementation detail. We can expose some more specific change events instead, or just listen to specific keys?

Reply:
I agree to not expose from configModel. But I would rather import and depend on the submodels explicitly. 

Some of our Ui elements need access to specific data from each of the bundle life cycle models (pre validate, post validate, remote tf state). If we add specific change events to configModel, it will devolve into events highly coupled with these submodels anyway. 

Having said that, we don't really need it here 😅